packages/collector/test/tracing/cloud/aws/s3/app.js

```diff
@@ -138,11 +138,12 @@ const S3Api = {
           break;
         case methods.PROMISE:
           promise = s3[operation](options).promise();
-          span = instana.currentSpan();
-          span.disableAutoEnd();
 
           promise
             .then(data => {
+              span = instana.currentSpan();
+              span.disableAutoEnd();
```

Comment:
Seems I missed those when reviewing S3 earlier, but the same applies: disableAutoEnd and manually ending spans should not be necessary anywhere in the S3 or DynamoDB tests.